src/github/pullRequestManager.ts

```diff
@@ -867,38 +872,47 @@ export class PullRequestManager {
 		return ret.data;
 	}
 
-	public async submitReview(pullRequest: PullRequestModel): Promise<Comment[]> {
+	public async submitReview(pullRequest: PullRequestModel, event='COMMENT', body?: string): Promise<void> {
 		const pendingReviewId = await this.getPendingReviewId(pullRequest);
 		const { mutate } = await pullRequest.githubRepository.ensure();
 
 		if (pendingReviewId) {
 			const { data } = await mutate({
 				mutation: queries.SubmitReview,
 				variables: {
-					id: pendingReviewId
+					id: pendingReviewId,
+					event,
+					body
 				}
 			});
 
-			return data.submitPullRequestReview.pullRequestReview.comments.nodes.map(parseGraphQLComment);
+			const submittedComments = data.submitPullRequestReview.pullRequestReview.comments.nodes.map(parseGraphQLComment);
+			_onDidSubmitReview.fire(submittedComments);
 		} else {
 			Logger.appendLine(`Submitting review failed, no pending review for current pull request: ${pullRequest.prNumber}.`);
 		}
 	}
 
 	async requestChanges(pullRequest: PullRequestModel, message?: string): Promise<any> {
-		return this.createReview(pullRequest, ReviewEvent.RequestChanges, message)
+		const action: Promise<any> = await this.getPendingReviewId(pullRequest)
+				? this.submitReview(pullRequest, 'REQUEST_CHANGES', message)
+				: this.createReview(pullRequest, ReviewEvent.RequestChanges, message);
+
+		return action

```

Comment:
It seems the result of `createReview` is not used anywhere, if that's true we don't need to return `any`.